gnd/src/main/java/com/google/android/gnd/persistence/sync/TileSourceDownloadWorker.java

```diff
@@ -209,8 +212,8 @@ public String getNotificationTitle() {
     return getApplicationContext().getString(R.string.downloading_tiles);
   }
 
-  static class TileDownloadException extends RuntimeException {
-    TileDownloadException(String msg, Throwable e) {
+  static class TileSourceDownloadException extends RuntimeException {
```

Comment:
Same comment here - the issue isn't that tiles couldn't be download _from_ the tile source, so I'd still expect this to be a `TileDownloadException`.